tests/platforms/web/static/test_run_dev.py

```diff
@@ -1,25 +1,124 @@
-import re
+from unittest import mock
 
 import pytest
 
-from briefcase.exceptions import UnsupportedCommandError
+from briefcase.exceptions import BriefcaseCommandError, UnsupportedCommandError
 from briefcase.platforms.web.static import StaticWebDevCommand
 
 
+class DummyStaticWebDevCommand(StaticWebDevCommand):
```

Comment:
The platform tests shouldn't be mocking the whole of the dev command - they should only be mocking the tools. Compare and contrast this test module with the dev tests for other platforms (or, for that matter, the existing tests in *this* module).